3.10/release-notes-known-issues310.md

```diff
@@ -67,4 +70,6 @@ Other
 | **Date Added:** 2019-04-03 <br> **Component:** arangod <br> **Deployment Mode:** Cluster <br> **Description:** Updating the properties of a collection in the cluster may return before the properties are updated consistently on all shards. This is especially visible when setting a schema for a collection with multiple shards, and then instantly starting to store non-conforming documents into the collection. These may be accepted until the properties change has been fully propagated to all shards. <br> **Affected Versions:** 3.7.x, 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** N/A |
 | **Date Added:** 2021-04-07 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** The Batch API (HTTP endpoint `/_api/batch`) cannot be used in combination with Stream transactions to submit batched requests, because the required header `x-arango-trx-id` is not forwarded. It only processes `Content-Type` and `Content-Id`. <br> **Affected Versions:** 3.5.x, 3.6.x, 3.7.x, 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#13552](https://github.com/arangodb/arangodb/issues/13552){:target="_blank"} |
-| **Date Added:** 2021-08-06 <br> **Component:** Installer <br> **Deployment Mode:** Single Server <br> **Description:** The Windows installer fails during database initialization with the error `failed to locate tzdata` if there are non-ASCII characters in the destination path. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-531](https://arangodb.atlassian.net/browse/BTS-531){:target="_blank"} (internal) |
\ No newline at end of file
+| **Date Added:** 2021-08-06 <br> **Component:** Installer <br> **Deployment Mode:** Single Server <br> **Description:** The Windows installer fails during database initialization with the error `failed to locate tzdata` if there are non-ASCII characters in the destination path. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-531](https://arangodb.atlassian.net/browse/BTS-531){:target="_blank"} (internal) |
+| **Date Added:** 2022-09-29 <br> **Component:** Web UI <br> **Deployment Mode:** All <br> **Description:** In the Web UI for ArangoSearch Views, it's not possible to create a link to a collection, if the collection has only one letter in its name. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-1008](https://arangodb.atlassian.net/browse/BTS-1008){:target="_blank"} (internal) |
```

Comment:
We should change all occurrences of "Web UI" to "web interface".
Does this refer to the traditional View type only?
```suggestion
| **Date Added:** 2022-09-29 <br> **Component:** Web UI <br> **Deployment Mode:** All <br> **Description:** In the web interface for Views of the `arangosearch` type, it's not possible to create a link to a collection, if the collection has only one letter in its name. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-1008](https://arangodb.atlassian.net/browse/BTS-1008){:target="_blank"} (internal) |
```